src/oas/oas-to-ms3/oas-30-to-ms3/index.ts

```diff
@@ -55,23 +55,42 @@ class MS3toOAS30toMS3 {
       version: info.version
     };
 
+    if (this.oasAPI.servers && this.oasAPI.servers.length && this.oasAPI.servers[0].url) {
+      settings.baseUri = this.oasAPI.servers[0].url;
+    }
+
     if (info.description) settings.description = info.description;
 
     return settings;
   }
 
   convertOperations(operations: OAS30Interface.PathItemObject | any): MS3Interface.Method[] {
     const methodsKeys = ['get', 'post', 'put', 'delete', 'options', 'head', 'patch'];
-
-    return methodsKeys.reduce((methodsArray: any[], methodKey: string) => {
+    const foundTraitsName = <any>[];
```

Comment:
why not foundTraitsName: string[] = [];

Reply:
[ts] 'string' only refers to a type, but is being used as a value here.